internal/shared/editor_helper.go

```diff
@@ -0,0 +1,49 @@
+package shared
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+var editor = "vim"
+
+func init() {
+	if s := os.Getenv("VISUAL"); s != "" {
```

Comment:
Use `VISUAL` in preference to `EDITOR` when available. See e.g. https://unix.stackexchange.com/questions/4859/visual-vs-editor-what-s-the-difference